app.rb

```diff
@@ -93,6 +106,7 @@ def create_a_book
     book = Book.new(title, author)
     @books << book
     p 'Book created sucessfully!'
+    save_books
   end
 
```

Comment:
![image](https://github.com/over-geek/OOP-School-Library/assets/93039359/8051527f-9235-42e5-ad56-6f7f39bac12d)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you rename your readme title because i don't think so it is Morse Code 😄 Kindly rename it to make your readme have correct data about your project 👍